docs/reference/release-notes/8.4.1.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[release-notes-8.4.1]]
+== {es} version 8.4.1
+
+coming[8.4.1]
```

Comment:
We'll need a follow-up PR to remove the `coming` tag when 8.4.1 is GA.

Reply:
I can open a PR. We still have some other stray coming tags it looks like.

Reply:
Thanks @debadair. I'll leave this as is for now.